src/data/rooms.json

```diff
@@ -4,11 +4,17 @@
     "title": "Webex Component User",
```

Comment:
The title for this space can be something else like `Test Room`. To differentiate between other group spaces and 1:1 spaces